fxa-client/sdks/android/library/src/main/java/org/mozilla/fxaclient/internal/Config.kt

```diff
@@ -8,41 +8,47 @@ package org.mozilla.fxaclient.internal
  * Config represents the server endpoint configurations needed for the
  * authentication flow.
  */
-class Config internal constructor(rawPointer: RawConfig) : RustObject<RawConfig>(rawPointer) {
+class Config internal constructor(val contentUrl: String, val clientId: String, val redirectUri: String) {
```

Comment:
I almost wonder if this should be a public ctor, in case someone wants to use another fxa stack (self hoster, local dev, or even `https://latest.dev.lcip.org`, for example).

At the very least it should be exposed as `Config.custom`, IMO.